backend/requirements.txt

```diff
@@ -0,0 +1,56 @@
+asn1crypto==0.24.0
```

Comment:
This file is not needed. If you want to add some requirements please do so in `backend/requirements/{base,dev}.txt`.

Reply:
Removed.